dart/lib/src/open_location_code.dart

```diff
@@ -16,7 +16,7 @@
 
```

Comment:
_This is an automated bot comment from the TravisCI tests_<br>**File has `dartanalyzer` errors that must be addressed**:<br>
<pre>Analyzing lib/src/open_location_code.dart...
  error • Const variables must be initialized with a constant value at lib/src/open_location_code.dart:86:29 • const_initialized_with_non_constant_value
1 error found.</pre>